src/diffusers/pipelines/flux/pipeline_flux_prior_redux.py

```diff
@@ -378,10 +449,10 @@ def __call__(
                 pooled_prompt_embeds,
                 _,
             ) = self.encode_prompt(
-                prompt=[""] * batch_size,
-                prompt_2=None,
-                prompt_embeds=None,
-                pooled_prompt_embeds=None,
+                prompt=prompt,
```

Comment:
let's throw out a warning here: 
if  `prompt` inputs is passed but do not have text_encoder/tokenizer, in this case the text inputs will be ignored 

it is a bit of different from our regular pipelines, normally, if you pass a prompt and do not have a text_encoder, you will get an error says like from encode_prompt; here we will just use zero prompt embeds instead, so let's be make an explicit warning about that

Reply:
agree, added one now